File: Data Structures & Algorithms/jump-game/submission-0.go
func canJump(nums []int) bool {
    n := len(nums)
    reachable := make([]bool, n)
    reachable[0] = true
    for i := 0; i < n; i++ {
        num := nums[i]
        if reachable[i] {
            for j := i+1; j <= i+num && j < n; j++{
                reachable[j] = true
            }
        }
    }
    return reachable[n-1]
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        n = len(nums)
        if n < 2:
            return n
        vals = set(nums)
        res = 0
        for num in vals:
            if (num-1) not in vals:
                cnt = 0
                while cnt <= n and (num+cnt) in vals:
                    cnt += 1
                if cnt > res:
                    res = cnt
        return res